spec/index.html

```diff
@@ -1149,12 +1601,300 @@ <h3>Overview</h3>
+          following quads:</p>
+
+        <pre id="ex-nd-shared-hashes-e1-quads" data-transform="updateExample">
+          <!--
+            :p :q _:e1 .
+            _:e1 :p _:e3 .
+          -->
+        </pre>
+
+        <p><a href="#hndq-3">Step 3</a> again calculates <a>blank node</a> <var>components</var>
+          using
+          <a href="#hash-related-blank-node">Hash Related Blank Node algorithm</a>.</p>
+
+        <table id="ex-nd-shared-table-related-hash-e1"
+               class="hash-related numbered">
+          <caption>Hash for `_:e3`  related to `_:e1`</caption>
+          <thead><tr>
+            <th><var>related</var></th>
+            <th><var>quad</var></th>
+            <th><var>position</var></th>
+            <th><var>hash</var></th>
+          </tr></thead>
+          <tbody>
+            <tr>
+              <td>`_:e3`</td>
+              <td>`_:e1 :p _:e3 .`</td>
+              <td>`o`</td>
+              <td>
+                <!-- canonical identifer _:e3 is _:c14n1 -->
+                <!-- hash "o<http://example.com/#p>_:c14n1"-->
+                <code>b7956ea1d654d5824496eb439a1f2b79478bd7d02d4a115f4c97cbff6b098216</code></td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p>which results in the following <var>H<sub>n</sub></var>:</p>
+
+        <table id="ex-nd-shared-table-bn-list-e1"
+               class="hash-related numbered">
+          <caption>Blank node list for hash</caption>
+          <thead><tr>
+            <th><var>related hash</var></th>
+            <th><var>blank node list</var></th>
+          </tr></thead>
+          <tbody>
+            <tr>
+              <td><code>b7956ea1d654d5824496eb439a1f2b79478bd7d02d4a115f4c97cbff6b098216</code></td>
+              <td>`[ _:e3 ]`</td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p><a href="#hndq-5">Step 5</a>
+          runs in essentially the same manner, mapping just `_:e3`
+          having the canonical identifier `_:c14n1`,
+          so there is again a single permutation
+          resulting in one candidate path.
+          The resulting <var>chosen path</var> is `_:c14n1`.</p>
+
+        <p>The string <var>data to hash</var> is composed
+          of the single <var>related hash</var> and the <var>chosen path</var>:
+          `af54b9512b1ef069205e8e41bc5a96e86a108b0389caa5029f2c3fd0bc465246_:c14n1`,
+          which hashes to
+          `767a5e66f509221f45003a16c12a89d4d9675cfa51ffa80459b63606bdfc2ada`.</p>
+
+        <p>The string <var>data to hash</var> is composed
+          of the single <var>related hash</var> and the <var>chosen path</var>:
+          `b7956ea1d654d5824496eb439a1f2b79478bd7d02d4a115f4c97cbff6b098216_:c14n1`,
+          which hashes to
+          `fbc300de5afafd97a4b9ee1e72b57754dcdcb7ebb724789ac6a94a5b82a48d30`,
+          which, along with the <a>identifier issuer</a> used to traverse
```

Comment:
In this context, `temporary issuer` might as well be more appropriate, while it is actually `identifier issuer` though.
```suggestion
          which, along with the <var>temporary issuer</var> used to traverse
```